package.json

```
@@ -36,7 +36,7 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@salvoravida/react-redux": "^7.1.8",
+    "@salvoravida/react-redux": "^7.1.9",
```

```suggestion
    "@salvoravida/react-redux": "^7.1.8",
```
CI is failing. Let me do this later.

---

Oh, can you commit the suggestion?

---

done